libs/sdk-ui-dashboard/api/sdk-ui-dashboard.api.md

```diff
@@ -20,105 +20,98 @@ import { DateFilterType } from '@gooddata/sdk-backend-spi';
 import { DateString } from '@gooddata/sdk-backend-spi';
 import { Dictionary } from '@reduxjs/toolkit';
 import { Dispatch } from '@reduxjs/toolkit';
-import { DrillDefinition } from '@gooddata/sdk-backend-spi';
+import { DrillDefinition } from '@gooddata/sdk-model';
 import { EntityId } from '@reduxjs/toolkit';
 import { EntityState } from '@reduxjs/toolkit';
 import { ExplicitDrill } from '@gooddata/sdk-ui';
-import { FilterContextItem } from '@gooddata/sdk-backend-spi';
-import { FilterContextItem as FilterContextItem_2 } from '@gooddata/sdk-model';
+import { FilterContextItem } from '@gooddata/sdk-model';
+import { FilterContextItem as FilterContextItem_2 } from '@gooddata/sdk-backend-spi';
```

Comment:
Seems there is some spi import left behind.

Reply:
Removed.